evalml/utils/gen_utils.py

```diff
@@ -270,3 +270,27 @@ def is_all_numeric(df):
         if dtype not in numeric_and_boolean_dtypes:
             return False
     return True
+
+
+def _convert_woodwork_types_wrapper(pd_data):
+    """
+    Converts a pandas data structure that may have extension or nullable dtypes to dtypes that numpy can understand and handle.
+
+    Arguments:
+        pd_data (pd.Series, pd.DataFrame, pd.ExtensionArray): Pandas data structure
+
+    Returns:
+        New pandas data structure (pd.DataFrame or pd.Series) with original data and dtypes that can be handled by numpy
+    """
+    nullable_to_numpy_mapping = {pd.Int64Dtype: 'int64',
+                                 pd.BooleanDtype: 'bool',
+                                 pd.StringDtype: 'object'}
+    if isinstance(pd_data, pd.api.extensions.ExtensionArray):
+        pd_data = pd.Series(pd_data)
+    if isinstance(pd_data, pd.Series) and type(pd_data.dtype) in nullable_to_numpy_mapping:
```

Comment:
@angela97lin is there a reason to do
```python
type(pd_data.dtype) in nullable_to_numpy_mapping
```
over
```python
pd_data.dtype in nullable_to_numpy_mapping
```
?

Reply:
There's a slight nuance. For example:
>> pd_data.dtype
Int64Dtype()
>> type(pd_data.dtype)
<class 'pandas.core.arrays.integer.Int64Dtype'>

So alternatively, I could update the `nullable_to_numpy_mapping` to hold instances (`pd.Int64Dtype()`) instead of classes (`pd.Int64Dtype`) to make this work. I don't have a preference for one over the other!